src/build.sh

```diff
@@ -75,9 +85,19 @@ while [[ $# -gt 0 ]]; do
   esac
 done
 
+# Interactive prompts if values are missing
```

Comment:
The build.sh script should fail if these do not have values, why are we adding these prompts?